core/server/src/test/java/alluxio/worker/block/BlockLockManagerTest.java

```diff
@@ -174,12 +173,11 @@ public void cleanupSessionTest() throws Exception {
   @Test(timeout = 10000)
   public void grabManyLocksTest() throws Exception {
     int maxLocks = 100;
-    setMaxLocks(maxLocks);
     BlockLockManager manager = new BlockLockManager();
     for (int i = 0; i < maxLocks; i++) {
       manager.lockBlock(i, i, BlockLockType.WRITE);
     }
-    lockExpectingHang(manager, 101);
+    lockExpectingHang(manager, 101, false);
```

Comment:
Actually we can delete this test since WORKER_TIERED_STORE_BLOCK_LOCKS doesn't exist any more.